tests/ha/multi_pod_failure/test_multi_server_pod_failure.py

```diff
@@ -1856,7 +1701,7 @@ def test_copy_object_during_kserver_pods_down(self):
             assert_utils.assert_true(False, "Failed to do copy object when cluster has "
                                             f"some failures.Failed buckets: {failed_bkts}")
         elif exp_fail_bkt_obj_dict:
-            LOGGER.info("Step 7.1: Retrying copy object to buckets %s",
+            LOGGER.info("Step 4.1: Retrying copy object to buckets %s",
```

Comment:
Do we need this as a step. There are no more steps inside 4 though

Reply:
Yes, we can keep this as it is, to have same code line with other files too (data pod failure, server pod failure, etc)